core/src/test/java/org/pentaho/platform/dataaccess/datasource/wizard/csv/FileUtilsTest.java

```diff
@@ -17,20 +17,26 @@
 
 package org.pentaho.platform.dataaccess.datasource.wizard.csv;
 
+import org.junit.Ignore;
```

Comment:
Is this import being used?

Reply:
Nope, I think I was ignoring some tests when fixing stuff and missed removing that, I'll update it